packages/reactive-element/src/decorators/query-assigned-elements.ts

```diff
@@ -93,5 +97,13 @@ export function queryAssignedElements(
         ) as V;
       },
     };
+    if (typeof name !== 'object' && extendedReflect.decorate) {
+      // If we're called as a legacy decorator, and Reflect.decorate is present
+      // then we have no guarantees that the returned descriptor will be
+      // defined on the class, so we must apply it directly ourselves.
+      Object.defineProperty(obj, name, descriptor);
+      return;
```

Comment:
is it bad to let this fall through and return the descriptor?

Reply:
Let's try it, might have to revert it internally, but we'll see